api/app/src/routes/middlewares/auth.ts

```diff
@@ -32,6 +33,11 @@ export const checkMAPIAccess = async (
   res: Response,
   next: NextFunction
 ): Promise<void> => {
+  // skip check on local env
+  if (!Config.isCloudEnv()) {
+    next();
+    return;
+  }
```

Comment:
At first glance this is good as it helps with testing w/o worrying to set access. But OTOH, it prevents us from testing an important feature which is access control to MAPI.

Since the work to give access is super simple*, maybe we leave as it was?

*On Postman: `POST /internal/mapi-access?cxId=xxx`

Reply:
I like the approach to pre-populate the dev DB with a known user, so we can quickly get it working. It's super useful in the case one needs a clean slate DB as well.

Reply:
will remove this for now and setup in a subsequent PR